### PR TITLE
Fixes duplicated module release

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

File: alws/release_planner.py [hunk 1]
```diff
@@ -215,10 +215,10 @@ async def prepare_data_for_executing_async_tasks(
             for repo in self.base_platform.repos:
                 pulp_repo_info = pulp_repos[repo.pulp_href]
                 self.repo_data_by_href[repo.pulp_href] = (repo.id, repo.arch)
-                is_debug = bool(re.search(r'debug(info|source|)',
-                                          pulp_repo_info['name']))
+                repo_is_debug = bool(re.search(r'debug(info|source|)',
```
Comment (soksanichenko) [inline alws/release_planner.py:218]: `re.search(...) is not None`

**Review by soksanichenko — APPROVED** [review]

File: alws/release_planner.py (hunk 1 above)
Comment (AvoidMe) [inline alws/release_planner.py:218]: Please, remove last `|`, since `name-debug` package is not a debug package.

File: alws/utils/modularity.py [hunk 2]
```diff
@@ -301,6 +301,10 @@ def iter_dependencies(self):
                 for stream in dep.get_buildtime_streams(module):
                     yield module, stream
 
+    def get_nsvca(self) -> str:
+        return (f'{self.name}:{self.stream}:{self.version}:'
+                f'{self.context}:{self.arch}')
```
Comment (AvoidMe) [inline alws/utils/modularity.py:306]: Please, make it property, just like the rest of attributes (name, stream, version, etc)

File: alws/release_planner.py (hunk 1 above)
Comment (anfimovdm) [reply alws/release_planner.py:218]: we looking here for debug repos, not packages, but i'll fix it anyway

Comment (AvoidMe) [reply alws/release_planner.py]: Sorry, was confused by previous change, than last `|` should stay

**Review by AvoidMe — APPROVED** [review]

**Pull request merged**